### DISCUSSION TITLE
Estou com esse erro "polling.js:298 GET http://file/socket.io/?EIO=4&transport=polling&t=OZpxKwH net::ERR_NAME_NOT_RESOLVED"

### DISCUSSION THREAD

**Discussion by JoaoDias12** [socketio/socket.io#4750] [Q&A] [answered]
estou criando um jogo usando electron.js e socket.io, adicionei recentemente o socket.io coloquei as dependências e etc... e ele esta funcionando em partes, quando dou node server.js ele diz que executou o server na porta 3000, e quando dou npm start e meu app abre, no terminal aparece o id e que um player conectou, porem no meu console fica dando esse erro varias vezes

**Comment by Santosl2** [comment, marked as answer]
Primeiramente: é necessário publicar sua dúvida em ingles.

Segundo: O URL do socket está errado, se você notar ele está batendo no http://file/ .
Você pode tentar desativar outros tipos de transport, e utilizar apenas o webosocket... 
Exemplo:

```typescript
io("http://localhost:3001", {
      autoConnect: false,
      transports: ["websocket"],
      upgrade: false,
    })
```
